packages/subgraph/tests/addresslist-voting/addresslist-voting.test.ts

```diff
@@ -323,6 +323,52 @@ test('Run AddresslistVoting (handleVoteCast) mappings with mock event', () => {
   clearStore();
 });
 
+test('Run AddresslistVoting (handleVoteCast) mappings with mock event and vote option "None"', () => {
+  // create state
+  let proposal = createAddresslistVotingProposalEntityState();
+
+  // create calls
+  createGetProposalCall(
+    CONTRACT_ADDRESS,
+    PROPOSAL_ID,
+    true,
+    false,
+
+    // ProposalParameters
+    VOTING_MODE,
+    SUPPORT_THRESHOLD,
+    MIN_PARTICIPATION,
+    START_DATE,
+    END_DATE,
+    SNAPSHOT_BLOCK,
+
+    // Tally
+    '0', // abstain
+    '0', // yes
+    '0', // no
+    TOTAL_VOTING_POWER,
+
+    actions
+  );
+
+  // create event
+  let event = createNewVoteCastEvent(
+    PROPOSAL_ID,
+    ADDRESS_ONE,
+    '0', // yes
```

Comment:
Does '0' mean that this is a "yes" vote?

Reply:
it means none. I updated the comment